docs/fastddsgen/pubsub_app/includes/first_app.rst

```diff
@@ -28,7 +18,7 @@ The `-example` option creates an example application, and the CMake files needed
 
             <path-to-Fast-DDS-workspace>/src/fastrtpsgen/scripts/fastrtpsgen -example CMake HelloWorld.idl
```

Comment:
I'm not sure, does `colcon` install this in the `install` dir?

Reply:
Those scripts do not need installation and are downloaded when cloning the repositories. 
https://github.com/eProsima/Fast-DDS-Gen/tree/master/scripts

Reply:
Sure, my question was more what happens when you `gradle assemble`. Do they get copied somewhere?